wpeverest/everest-forms
# Feature  - Dashboard page

### All Submissions:


### Changes proposed in this Pull Request:

This PR gives the new dashboard page which shows details about our plugin, products, and many more sections.

### How to test the changes in this Pull Request:
1. Switch to the branch and open the terminal and run `npm run dev ` or `npm run build`. If any error occurs please resolve the issues and run again.
2. Go to Everest Forms -> Dashboard.
3. Check if every feature inside the Dashboard page is working properly.
4. Check if the Setup Wizard and Everest Forms Blocks are working properly as the build of this feature may hamper 

### Types of changes:


### Other information:



### Changelog entry

> Feature  - Dashboard page.
